misc/protobuf-ballerina/src/main/java/org/ballerinalang/protobuf/utils/BalFileGenerationUtils.java

```diff
@@ -104,6 +105,24 @@ public static String getDescriptorPath(String protoPath) {
                         BalGenerationConstants.EMPTY_STRING) + DESC_SUFFIX;
     }
     
+    /**
+     * Resolve proto folder path from Proto file path
+     *
+     * @param protoPath Proto file path
+     * @return
+     */
+    public static String resolveProtoFloderPath(String protoPath) {
+        int idx = protoPath.lastIndexOf
+                (BalGenerationConstants.FILE_SEPARATOR);
```

Comment:
This line can be merged with the previous line.

Reply:
Fixed in f2b7cd26cb71721658fbc018e48d1aecaddaecd2